apps/src/code-studio/pd/foorm/FoormEditor.jsx

```diff
@@ -193,33 +199,187 @@ class FoormEditor extends React.Component {
       .done(result => {
         this.setState({
           lastValidated: moment().format(TIME_FORMAT),
-          validationError: false
+          validationError: false,
```

Comment:
Nit, but could this be a blank string by default? Kind of implies a true/false value but actually gets set to the value of the error message, right?

Reply:
set to null instead since I am using it to check if there is a validation error